extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -69,7 +70,7 @@ class AnalyticsEvent {
 
   /**
    * @param {!AnalyticsEventType} type The type of event.
-   * @param {!Object<string, string>} A map of vars and their values.
+   * @param {!Object<string, string>} [vars] A map of vars and their values.
```

Comment:
We tend to use the "Google Closure Compiler" syntax. In this particular case, I think the notation would be:

`{Object<string, string>} opt_vars` and change the name of the parameter from `vars` to `opt_vars`.